SPECS/rust/rust.signatures.json

```
@@ -2,11 +2,11 @@
  "Signatures": {
   "cargo-1.58.0-aarch64-unknown-linux-gnu.tar.gz": "68e2e7493af55c5792636c2b2b0b497b1e43b12eb2e91da71e92426701884c24",
   "cargo-1.58.0-x86_64-unknown-linux-gnu.tar.gz": "940aa91ad2de39c18749e8d789d88846de2debbcf6207247225b42c6c3bf731a",
-  "rust-1.59.0-cargo.tar.gz": "8d2076b86d6cafe4181bf0738cc52e4490c5c0bcffffd86c087dfc710acae198",
+  "rust-1.59.0-cargo.tar.gz": "59a7bfc0e959e4b92ec54dc53a7a59c192baf50852813b5e58ea3b0f2f4a883f",
```

Making this suggestion since I uploaded the next tar file with this signature
```suggestion
  "rust-1.59.0-cargo.tar.gz": "04a6b4637ae8b1b5470b2e36e990b54bc5a8978b1d0042db8c1cff5710474f04",
```